week1/homework/src/server.js

```diff
@@ -1,17 +1,40 @@
 'use strict';
 
+const url = require('url');
 const http = require('http');
+// const fs = require('fs');
 
 /* `createServer` MUST return an instance of `http.Server` otherwise the tests
  * will fail.
  */
 function createServer(port) {
   let state = 10;
+  let newState = state;
```

Comment:
this is amazing Rabia, and you're the only one to do this so far.
it's very important to not mutate the state, we will see why in the next lessons :)